x-pack/solutions/security/plugins/elastic_assistant/public/src/hooks/is_nav_control_visible/use_is_nav_control_visible.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useEffect, useState } from 'react';
+import { combineLatest } from 'rxjs';
+import { DEFAULT_APP_CATEGORIES, type PublicAppInfo } from '@kbn/core/public';
+import { AIAssistantType } from '@kbn/ai-assistant-management-plugin/public';
+import { useKibana } from '../../context/typed_kibana_context/typed_kibana_context';
+
+function getVisibility(
+  appId: string | undefined,
+  applications: ReadonlyMap<string, PublicAppInfo>,
+  preferredAssistantType: AIAssistantType
+) {
+  // The "Global assistant" stack management setting for the security assistant still needs to be developed.
+  // In the meantime, while testing, show the Security assistant everywhere except in Observability.
+
+  const categoryId =
+    (appId && applications.get(appId)?.category?.id) || DEFAULT_APP_CATEGORIES.kibana.id;
+
+  return ![DEFAULT_APP_CATEGORIES.observability.id].includes(categoryId);
```

Comment:
Will want to exclude from the search app as well

<p align="center">
  <img width="500" src="https://github.com/user-attachments/assets/ef335928-a617-428e-a977-544c4bc7f764" />
</p>